modules/ROOT/pages/application-deployment.adoc

```diff
@@ -4,11 +4,15 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: deploy, application, applications, runtime, amc, cloudhub, on premises, on premise
 
+Learn how to deploy, undeploy and update Mule runtime engine (Mule) applications.
+
 == Starting Mule
 
-Start Mule by running `<MULE_HOME>/bin/mule` or starting Mule as a service. By default, applications in the `<MULE_HOME>/apps` directory are deployed. You can also designate specific applications to start (separated by a colon - **:**) and Mule 4 respects the order when starting the applications. In this scenario, only the specified applications start.
+Start Mule from a command line tool by running `$MULE_HOME/bin/mule` or start Mule as a service. By default, applications in the `$MULE_HOME/apps` directory are deployed.
```

Comment:
By default, applications > By default **all** applications.
I'd suggest adding **all** to emphasize the contrast of specifying mule.deploy.applications to start specific apps vs. not using the property to start all apps.

Reply:
Updated.